cc-snapshot

```diff
@@ -69,6 +66,38 @@ while getopts ":hp:e:f" opt; do
   esac
 done
 
+###### Update section ######
+
+#Download latest version, assign versions to variables, curlstat variable ensures update only executes if curl succeeds.
+#------------------------------------------------
+SCRIPT=$0
+curl -s -o /tmp/tmp-snap https://raw.githubusercontent.com/ChameleonCloud/cc-snapshot/fhlb_test/cc-snapshot
+CURLSTAT=$?
+NEWSCRIPT='/tmp/tmp-snap'
+
+#Compare files and update if cc-snapshot outdated
+#------------------------------------------------
+diff -q "$SCRIPT" "$NEWSCRIPT" &>/dev/null
+DIFF=$?
+if [ $DIFF -eq 1 ] && [ $CURLSTAT -eq 0 ]  && [ $IGNORE_WARNING == false ];
+then
+        echo 'Your cc-snapshot is not up to date.'
+        read -ep 'Update now? [y/n]'
```

Comment:
It would be better to add the check for IGNORE_WARNING above this line, so that non-interactive users still get the warning.

Reply:
Thank you!!